### DISCUSSION TITLE
Anomalia zigbee dopo passaggio a SSD

### DISCUSSION THREAD

**Discussion by Stomodine** [indomus/forum#1693] [ZigBee, Z-Wave, Thread, Matter]
ciao

riassumo brevemente la situazione: Raspberry Pi4 con Hassio e alimentatore originale rasp , scheda SD e Conbee II gestito da Zigbee2mqtt; funziona tutto a meraviglia con circa 10 componenti associati quando ad un certo punto si fotte la scheda SD.
Decido quindi di passare ad un SSD M.2 e avendo installato il google drive backup recupero quello di un paio di settimane prima e faccio il restore su SSD; da allora ho continui problemi con i dispositivi zigbee nel senso che a volte li vedo offline,a volte non vedo più l'entità su lovelace e a volte non aggiornano i valori in tempo reale.
Ho fatto varie prove tra cui "ritornare" ad Hassio su scheda SD e lì funziona tuttto perfettamente quindi pare che ci sia qualche problema con l'hard disk; tenendo conto delle considerazioni seguenti qualcuno ha qualche idea?
-SSD installato su USB 2.0 (e non 3.0 che potrebbe dare interferenze)
-Conbee installata su USB 2.0 e prolunga messa lontana dall'hard disk
-Usato un Hard disk diverso,stessi problemi
-Quando cerco di caricare la mappa su zigbee2mqqt va in idle circa 2 minuti e poi torna sul pulsante "carica mappa" (su SD caricava senza problemi)
-Provato a disaccoppiare un dispositivo e riaccoppiarlo: è stato un parto nel senso che avrò provato 10 volte e all'undicesima è andato a caso
-Provato a cambiare canale da 11 a 12 ma ha peggiorato la situazione (tutte le entità erano offline)
-Quando apro il canale per un nuovo accoppiamento si Z2M ci mette qualche secondo ma quando lo chiudo passano 2 minuti buoni (cosa che con SD non faceva)

**Comment by snakuzzo** [comment]
Difficile dire da cosa possa dipendere.
Io posso assicurarti che utilizzando un SSD esterno su un RPI funziona tutto lo stesso.

**Reply by Stomodine** [reply]
l'ultima prova che farò è quella di mettere un hub usb alimentato
https://forum.phoscon.de/t/solved-conbee-ii-stopped-adding-devices/193/13
poi sono col culo per terra

**Reply by snakuzzo** [reply]
Facci sapere

**Reply by Stomodine** [reply]
Spippolando in rete ho trovato un tizio che consigliava di disinstallare l'integrazione ZHA in quanto poteva andare in conflitto con Z2M; provato a rimuovere e tutto si è messo a funzionare alla perfezione.
Fin qui tutto fantastico peccato che reinserendo la SD ho visto che funzionava tutto anche con ZHA installato.
Hai qualche spiegazione?

**Reply by snakuzzo** [reply]
Scusa eh...ma tu innanzi tutto devi chiarirti un attimo delle cose.
ZHA, così come Z2M, hanno bisogno di un coordinator HW per funzionare...ok ?
Ora, in un'intallazione classica e salvo particolari esigenze, se ne tiene attivo uno solo.
Se proprio vuoi utilizzarne due, devono appoggiarsi ad un HW diverso...ovvero...due coordinator HW diversi.

**Reply by snakuzzo** [reply]
Se invece hai tutto configurato correttamente, il problema forse è legato proprio alle porte USB (da verificare anche l'assegnazione corretta delle porte...e nel caso utilizzare `udev`)

**Reply by Stomodine** [reply]
sul fatto che ci voglia un coordinator hw siamo tutti daccordo; al netto del fatto che non ho idea del perchè abbia installato anche ZHA (sono passati mesi dal commissioning di HA) mi resta il dubbio del perchè con SSD vanno in conflitto mentre con SD no.
è forse il discorso delle porte USB?
